config/redirects.json

```diff
@@ -2213,5 +2213,10 @@
         "source": "/videos/twitch/parsing-gdelt-web-data---mongodb-world-hackathon-series/",
         "destination": "/videos/",
         "permanent": true
+    },
+    {
+        "source": "/products/atlas/cloudflare-worker-rest-api-realm/",
+        "destination": "/products/atlas/cloudflare-worker-rest-api/",
```

Comment:
Just flagging that the article slug would need to be updated in Strapi first before this is deployed. Currently /products/atlas/cloudflare-worker-rest-api/ is going to 404.